content/docs/user-guide/components/reference/terrain/surface-material-list.md

```diff
@@ -5,8 +5,19 @@ description: 'Open 3D Engine (O3DE) Terrain Surface Materials List component ref
 weight: 100
 ---
 
-The **Terrain Surface Materials List** component defines mappings between surface types and render materials.  You can use this to change the appearance of a surface type within different areas of your game. When you assign a material to a surface type, all the visible surfaces of that type, within the bounds of the required [Axis Aligned Box Shape](/docs/user-guide/components/reference/shape/axis-aligned-box-shape/) component, will adopt that material.
+The **Terrain Surface Materials List** component defines mappings between surface types and render materials. You can use this to change the appearance of a surface type within different areas of your game. When you assign a material to a surface type, all the visible surfaces of that type, within the bounds of the required [Axis Aligned Box Shape](/docs/user-guide/components/reference/shape/axis-aligned-box-shape/) component, will adopt that material.
 
+This component makes use of [macro materials](/docs/user-guide/components/reference/terrain/terrain-macro-material) and [detail materials](/docs/user-guide/components/reference/terrain/terrain-detail-material/) across a large terrain surface and blends between them. This blending of detail materials with the macro material enables you to use small, high fidelity, tiled detail materials that have variations in color and lighting across the terrain.
```

Comment:
Minor suggestions to condense the wording.

Whether to use commas after multiple adjectives or not depends on what the adjectives apply to. If they all apply to the noun, then add the commas. (To determine this, you should be able to swap the adjectives around without changing the meaning, for example "tiled, high-fidelity, small detail materials".)
Otherwise, as per mbalfour-amzn's suggestion, each adjective describes the following adj+noun unit, so order matters, and you can remove the commas. 
https://style.mla.org/coordinate-adjectives-commas/

```suggestion
This component uses [macro materials](/docs/user-guide/components/reference/terrain/terrain-macro-material) and [detail materials](/docs/user-guide/components/reference/terrain/terrain-detail-material/) across a large terrain surface and blends between them. This blending of detail materials with the macro material enables you to use small high-fidelity tiled detail materials that have variations in color and lighting across the terrain.
```

Reply:
Sounds good! Have incorporated the "makes use of" -> "uses" change and took Mike's suggested change for the other part.

Reply:
Ooh, thanks for giving me the formal terminology to describe this! It's on the edge, but I _think_ "small" is a coordinate adjective in this case, so I think it reads better without the commas. That said, the entire sentence is awkward and I don't think we're conveying the idea quite right yet.

Maybe:
"By using macro materials for the large-scale low-fidelity color variations and tiling detail materials for the small-scale high-fidelity details, the resulting terrain has a much broader variety of color and texture than what can be achieved with just the detail materials by themselves."